x/stakeibc/keeper/msg_server_submit_tx.go

```diff
@@ -500,10 +500,6 @@ func (k Keeper) SubmitCalibrationICQ(ctx sdk.Context, hostZone types.HostZone, v
 	if hostZone.DelegationIcaAddress == "" {
 		return errorsmod.Wrapf(types.ErrICAAccountNotFound, "no delegation address found for %s", hostZone.ChainId)
 	}
-	validator, _, found := GetValidatorFromAddress(hostZone.Validators, validatorAddress)
```

Comment:
I think we should actually leave this in, but ignore the returned validator
```go
_, _, found := GetValidatorFromAddress(hostZone.Validators, validatorAddress)
if !found {
	return errorsmod.Wrapf(types.ErrValidatorNotFound, "no registered validator for address (%s)", validatorAddress)
}
```

That way the tx fails if someone specifies a validator address that's not in our set

Reply:
Done [here](https://github.com/Stride-Labs/stride/pull/950/commits/06144f37cf93f7317781c37a82952089bba21d08).